examples/next-typescript/pages/profile-ssr.tsx

```diff
@@ -3,7 +3,19 @@ import Layout from "../components/Layout";
 import withSession from "../lib/session";
 import PropTypes from "prop-types";
 
-const SsrProfile = ({ user }) => {
+type User = {
```

Comment:
In a real world application, wouldn't this type go into a folder like `@customTypes/User.ts`? And then `import type { User } from "@customTypes/User"`?

Or something similar. The idea being to have the type declared in a single file and import it everywhere we need it (profile ssr, userUser).

Reply:
done!